Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.xaml

```diff
@@ -11,7 +11,6 @@
         <Setter Property="TextWrapping" Value="Wrap" />
         <Setter Property="TextLineBounds" Value="TrimToBaseline" />
         <Setter Property="OpticalMarginAlignment" Value="TrimSideBearings" />
-        <Setter Property="FontSize" Value="18" />
```

Comment:
For compat sake, please add a setter for HeaderContentFontSize and TextContentFontSize and set it to 18